frontend/TestRun/ResultsTab.svelte

```diff
@@ -0,0 +1,122 @@
+<script>
+    import {onMount} from "svelte";
+
+    let fetching = true;
+    export let id = "";
+    export let test_id = "";
+    let results = [];
+
+
+    const fetchResults = async function () {
+        fetching = true;
+        try {
+            let apiResponse = await fetch(`/api/v1/run/${test_id}/${id}/fetch_results`);
+            let apiJson = await apiResponse.json();
+            if (apiJson.status === "ok") {
+
+                fetching = false;
+                results = apiJson.tables;
+                console.log(results);
+            }
+        } catch (error) {
+            console.log(error);
+        }
+    };
+
+    const durationToStr = (value) => {
+        const hours = Math.floor(value / 3600);
+        const minutes = Math.floor((value % 3600) / 60);
+        const seconds = value % 60;
+        return `${hours.toString().padStart(2, "0")}:${minutes.toString().padStart(2, "0")}:${seconds.toString().padStart(2, '0')}`;
+    };
```

Comment:
We have [humanize-duration](https://www.npmjs.com/package/humanize-duration) as a dependency, you can use that instead.